### PR TITLE
Sets limit-request-field_size for gunicorn to 16k

### PR DESCRIPTION
**Pull request by petrblaho**
## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

**Comment by petrblaho** [comment]
EVNT-692

File: run_app.sh [hunk 1]
```diff
@@ -26,13 +26,18 @@ if [ -z "$ACG_CONFIG" ]; then
     then PORT=8080
   fi
 
-  prometheus_multiproc_dir=$TEMPDIR gunicorn wsgi -w $NUM_WORKERS --threads $THREADS -b 0.0.0.0:$PORT --log-level=$LOG_LEVEL --access-logfile=- --config ./gunicorn.conf.py
+  if [ -z "$GUNICORN_REQUEST_FIELD_LIMIT" ];
+    then GUNICORN_REQUEST_FIELD_LIMIT=16380
+  fi
+
+  prometheus_multiproc_dir=$TEMPDIR gunicorn wsgi -w $NUM_WORKERS --threads $THREADS -b 0.0.0.0:$PORT --log-level=$LOG_LEVEL --limit-request-field_size=$GUNICORN_REQUEST_FIELD_LIMIT --access-logfile=- --config ./gunicorn.conf.py
```
Comment (vkrizan) [inline run_app.sh:33]: A bit unrelated to these changes, but the variables should be quoted.

File: run_app_locally.sh [hunk 2]
```diff
@@ -26,7 +26,11 @@ if [ -z "$ACG_CONFIG" ]; then
     then PORT=8080
   fi
 
-  prometheus_multiproc_dir=$TEMPDIR gunicorn wsgi -w $NUM_WORKERS --threads $THREADS -b 0.0.0.0:$PORT --log-level=$LOG_LEVEL --access-logfile=- --config ./gunicorn.conf.py
+  if [ -z "$GUNICORN_REQUEST_FIELD_LIMIT" ];
+    then GUNICORN_REQUEST_FIELD_LIMIT=16380
+  fi
+
+  prometheus_multiproc_dir=$TEMPDIR gunicorn wsgi -w $NUM_WORKERS --threads $THREADS -b 0.0.0.0:$PORT --log-level=$LOG_LEVEL --limit-request-field_size=$GUNICORN_REQUEST_FIELD_LIMIT --access-logfile=- --config ./gunicorn.conf.py
```
Comment (vkrizan) [inline run_app_locally.sh:33]: DRY

**Review by vkrizan — APPROVED** [review]
LGTM, except the lack of quotation.

**Pull request merged**